src/bundle/Resources/views/themes/admin/account/register/index.html.twig

```diff
@@ -20,7 +20,6 @@
         {{ form_widget(form.children.register, {
             attr: { class: 'btn ibexa-btn ibexa-btn--primary ibexa-login__btn ibexa-login__btn--register' }
         }) }}
-        {{ form_widget(form) }}
```

Comment:
+1 if it's removed as redundant instruction / -1 if you want to prevent rendering left over form fields (form_end call is missing render_rest option).

Reply:
I would say the first one, as all content type additional fields are rendered in the first loop. That would create problems if someone extends just the form with for example some Agreement Checkboxes, but in that case our registration action does not cover it as well. 
@adamwojs 

But if you have some other solution in mind I am open for it.

Reply:
It's fine. `form_end` will skip already rendered fields.